content/en/containers/kubernetes/configuration.md

```diff
@@ -68,84 +88,538 @@ agents:
     create: true
 ```
 
+[1]: /containers/cluster_agent
+
 {{% /tab %}}
 {{< /tabs >}}
 
 For DaemonSet configuration, see [DaemonSet Cluster Agent event collection][14].
 
+## Enable NPM collection
+
+{{< tabs >}}
+{{% tab "Datadog Operator" %}}
+
+In your `datadog-agent.yaml`, set `features.npm.enabled` to `true`.
+
+```yaml
+apiVersion: datadoghq.com/v2alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  global:
+    credentials:
+      apiKey: <DATADOG_API_KEY>
+
+  features:
+    npm:
+      enabled: true
+```
+
+Then apply the new configuration:
+
+```shell
+kubectl apply -n $DD_NAMESPACE -f datadog-agent.yaml
+```
+
+{{% /tab %}}
+{{% tab "Helm" %}}
+
+Update your `datadog-values.yaml` with the following configuration:
+
+```yaml
+datadog:
+  # (...)
+  networkMonitoring:
+    enabled: true
+```
+
+Then upgrade your Helm chart:
+
+```shell
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> datadog/datadog
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+For more information, see [Network Performance Monitoring][18].
+
+## Enable log collection
+
+{{< tabs >}}
+{{% tab "Datadog Operator" %}}
+In your `datadog-agent.yaml`, set `features.logCollection.enabled` and `features.logCollection.containerCollectAll` to `true`.
+
+```yaml
+apiVersion: datadoghq.com/v2alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  global:
+    credentials:
+      apiKey: <DATADOG_API_KEY>
+
+  features:
+    logCollection:
+      enabled: true
+      containerCollectAll: true
+```
+
+Then apply the new configuration:
+
+```shell
+kubectl apply -n $DD_NAMESPACE -f datadog-agent.yaml
+```
+
+{{% /tab %}}
+{{% tab "Helm" %}}
+Update your `datadog-values.yaml` with the following configuration:
+
+```yaml
+datadog:
+  # (...)
+  logs:
+    enabled: true
+    containerCollectAll: true
+```
+
+Then upgrade your Helm chart:
+
+```shell
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> datadog/datadog
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+For more information, see [Kubernetes log collection][17].
+
+## Enable process collection
+
+{{< tabs >}}
+{{% tab "Datadog Operator" %}}
+In your `datadog-agent.yaml`, set `features.liveProcessCollection.enabled` to `true`.
+
+```yaml
+apiVersion: datadoghq.com/v2alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  global:
+    credentials:
+      apiKey: <DATADOG_API_KEY>
+
+  features:
+    liveProcessCollection:
+      enabled: true
+```
+
+Then apply the new configuration:
+
+```shell
+kubectl apply -n $DD_NAMESPACE -f datadog-agent.yaml
+```
+
+{{% /tab %}}
+{{% tab "Helm" %}}
+Update your `datadog-values.yaml` with the following configuration:
+
+```yaml
+datadog:
+  # (...)
+  processAgent:
+    enabled: true
+    processCollection: true
+```
+
+Then upgrade your Helm chart:
+
+```shell
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> datadog/datadog
+```
+{{% /tab %}}
+{{< /tabs >}}
+
```

Comment:
Did you also want to link to https://docs.datadoghq.com/infrastructure/process/ page to keep the same format/flow as the other sections that link to their relative functionality pages ? 
i.e. "For more information, see [Live Processes](https://docs.datadoghq.com/infrastructure/process/).